source/css/scss/app/sitefooter.scss

```diff
@@ -45,7 +45,6 @@ $footer-top-bottom-padding: rem-calc(40);
 }
 
 .siteFooter-group:last-of-type {
-	position: absolute;
```

Comment:
I assume you have taken this out with good reason - I just want to reconfirm that this change doesn't break anything.  Same with the removed float from line 166.